.github/copilot-instructions.md

```diff
@@ -0,0 +1,32 @@
+## General
+
+* Make only high confidence suggestions when reviewing code changes.
+* Always use the latest version C#, currently C# 13 features.
```

Comment:
14?

Reply:
I know we don't have a concept of a preview for C# versions but I figured keeping it to versions of the language that have already been publicly shipped with SDK support would be safer and give it a reference to language features that are less likely to change through iterations?